app/carbonapi/http_handlers.go

```diff
@@ -759,7 +762,7 @@ func (app *App) resolveGlobs(ctx context.Context, metric string, useCache bool,
 
 func (app *App) getRenderRequests(ctx context.Context, m parser.MetricRequest, useCache bool,
 	toLog *carbonapipb.AccessLogDetails) ([]string, error) {
-	if app.config.AlwaysSendGlobsAsIs {
+	if app.config.ResolveGlobs == 0 {
```

Comment:
should this be `<= 0`?

Reply:
no, this one should be `== 0`, I just commented it above. :P

Reply:
I guess intended behaviour is a bit unclear atm cause PR description doesn't match explanation in comments. Let's wait for those to sync up and then prolly this thread will be clearer.